File: src/main/java/fyordo/lifeagragator/med/tag/dto/TagDto.java
package fyordo.lifeagragator.med.tag.dto;

import fyordo.lifeagragator.med.tag.Tag;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class TagDto {
    private Long id;
    private String title;
    private String color;

    public TagDto(Tag tag){
        id = tag.getId();
        title = tag.getTitle();
        color = tag.getColor();
    }
}

File: src/main/kotlin/fyordo/lifeagragator/med/tag/dto/TagDto.kt
package fyordo.lifeagragator.med.tag.dto

import fyordo.lifeagragator.med.tag.Tag

data class TagDto(
    var id: Long? = null,

    var title: String? = null,

    var color: String? = null,

    var textColor: String? = null
){
    constructor(tag: Tag) : this() {
        this.id = tag.id
        this.title = tag.title
        this.color = tag.color
        this.textColor = tag.textColor
    }
}
